### PR TITLE
28944 update coe app UI

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
This pull request updates some of the content for the COE Eligibility App. Updates include success, pending, and rejection states. This PR doesn't include document upload functionality, which will be included in a subsequent PR.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#28944

## Testing done
- local

## Screenshots
<details><summary>Eligibility States</summary>

Screen Shot 2021-09-02 at 2 30 41 PM
Screen Shot 2021-09-02 at 2 53 27 PM

Screen Shot 2021-09-02 at 2 59 57 PM
</details>

## Acceptance criteria
- [x] States are updated

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/lgy/coe/components/CoeDocumentUpload.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (micahchiang) [inline src/applications/lgy/coe/components/CoeDocumentUpload.jsx:1]: This file isn't fully functional yet. I realized the work to finish a document upload feature for this app should be in its own pull request.

**Review by cohnjesse — APPROVED** [review]
LGTM

**Review by ashagross — APPROVED** [review]
